resources/views/includes/elements/exhibitions/files.blade.php

```diff
@@ -1,5 +1,11 @@
-@section('exhibitions-files')
-    @if(!empty($exhibition['exhibition_files']))
+@if (!empty($page_template))
+    @if (!empty($exhibition['exhibition_files']))
         <x-exhibition-files :files="$exhibition['exhibition_files']"></x-exhibition-files>
     @endif
-@endsection
+@else
+    @section('exhibitions-files')
+        @if (!empty($exhibition['exhibition_files']))
+            <x-exhibition-files :files="$exhibition['exhibition_files']"></x-exhibition-files>
+        @endif
+    @endsection
+@endif
```

Comment:
Hi @ostafinskim , instead of the fix here, ahve you tried adding the `exhibitions-files` section to the `details-2024.blade.php` template? It feels like that would be a simpler solution if possible.

Reply:
I did but couldn’t get this part of UI within new template for some reason.